newsroom/companies/utils.py

```diff
@@ -23,3 +25,12 @@ def get_company_product_ids(company: Company) -> List[ObjectId]:
         ],
         key=lambda o: str(o),
     )
+
+
+def load_countries_list():
+    with open(get_filepath("vocabularies.json")) as f:
```

Comment:
I'm not sure it's good to load a file on every `GET` request to this view, especially when this file won't change during the runtime of the environment.

@petrjasek What are your thoughts on finally having a `vocabularies` resource in Newshub? Or should we load this file in the config, allowing app repo's to override?

Reply:
Also, there is no `vocabularies.json` file included in this PR

Reply:
here we are using superdesk-core `vocabularies.json` file.

Reply:
I don't see much use for it to be editable in newsroom atm, but maybe some way to define what metadata should be stored for companies/users eventually might require that. for now I would keep it static